.github/workflows/pre-commit.yml

```diff
@@ -17,4 +17,5 @@ jobs:
         python-version: |
           3.9
           3.x
+        cache: 'pip'
```

Comment:
```suggestion
```

`pre-commit/action` already has it's own caching, better not mix those.

Reply:
as discussed on chat - I think the installation of the pre-commit itself is not cached. If someone can confirm that's not the case, please do.

Reply:
It is cached: https://github.com/pre-commit/action/blob/f7acafac0271bdd064cdfa1b13f17b4350e565ed/action.yml#L15-L18

The Python environment is not cached, but that is not cached by `setup-python` either.

Reply:
looks like 10MB.  
> Installing collected packages: distlib, pyyaml, platformdirs, nodeenv, identify, filelock, cfgv, virtualenv, pre-commit

Reply:
By impact I was more referring to the time impact. Though it's not like it's a blocker for any of the runners anyway, so I guess it's ok with and without.

Reply:
Looks like no impact time-wise.

Reply:
ok I did a few test runs and it seems cache in pre-commit doesn't really make sense. Removed.